screeps-game-api/src/game.rs

```diff
@@ -202,73 +202,34 @@ pub mod shard {
     }
 }
 
-macro_rules! game_map_access {
-    ($(
-            #[$attr:meta]
-    )*
-    $mod_name:ident, $type:path, $js_inner:expr) => (
-        $(
-                #[$attr]
-        )*
-        pub mod $mod_name {
-            use objects;
-
-            /// Retrieve the string keys of this object.
-            pub fn keys() -> Vec<String> {
-                js_unwrap!(Object.keys($js_inner))
-            }
-
-            /// Retrieve all values in this object.
-            pub fn values() -> Vec<$type> {
-                js_unwrap!(Object.values($js_inner))
-            }
-
-            /// Retrieve a specific value by key.
-            pub fn get(name: &str) -> Option<$type> {
-                js_unwrap!($js_inner[@{name}])
-            }
-        }
-    );
-    ($(
-        $(
-            #[$attr:meta]
-        )*
-        ($mod:ident, $type:path, $js:expr)
-    ),* $(,)*) => {
-        $(
-            game_map_access!($(#[$attr])* $mod, $type, $js);
-        )*
-    };
-}
-
 game_map_access! {
     /// See [http://docs.screeps.com/api/#Game.constructionSites]
     ///
     /// [http://docs.screeps.com/api/#Game.constructionSites]: http://docs.screeps.com/api/#Game.constructionSites
-    (construction_sites, objects::ConstructionSite, Game.constructionSites),
+    construction_sites, objects::ConstructionSite, Game.constructionSites;
```

Comment:
I think we should keep macro syntax changes separate from the reorganization, at least for good history.

If we want to do some normalization on macro usage syntax, that would be a good idea, but I think it would be more useful to do and discuss in a new PR.